core/src/main/scala/kafka/log/Log.scala

```diff
@@ -2040,9 +2045,11 @@ class Log(@volatile private var _dir: File,
               localLogStartOffset = math.min(targetOffset, localLogStartOffset),
               endOffset = targetOffset
             )
+            offsetTruncatedTo = activeSegmentTruncationInfo.offsetTruncatedTo
```

Comment:
can we move these 2 lines to be immediately after line 2039 to make the logic more consodilated?